src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <div class="card__wrapper">
+        <img src="images/imac.jpeg"
+          alt="iMac" 
+          class="card__image"
+        >
+    
+        <div class="card__title-wrapper">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__title-label">Product code: 195434</p>
+        </div>
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star">
+            </div>
+
+            <div class="stars__star">
+            </div>
+
+            <div class="stars__star">
+            </div>
+
+            <div class="stars__star">
+            </div>
+            
+            <div class="stars__star">
+            </div>
+          </div>
```

Comment:
This could be written shorter.
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```

Reply:
done